api_app/analyzers_manager/observable_analyzers/dns0_names.py

```diff
@@ -0,0 +1,212 @@
+import re
+from logging import getLogger
+from typing import Dict
+from urllib.parse import urlparse
+
+import requests
+from dateutil import parser as dateutil_parser
+
+from api_app.analyzers_manager import classes
+from api_app.analyzers_manager.exceptions import (
+    AnalyzerConfigurationException,
+    AnalyzerRunException,
+)
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = getLogger(__name__)
+
+_supported_fuzzy_params = [
```

Comment:
local global variables should be declared with capital letters in python

Reply:
but nvm, I know that you copied that from the dnsdb.py file

Reply:
yes, also in many other files is written in lowercase